test/sales_engine_test.rb

```diff
@@ -104,7 +140,7 @@ def test_acceptance_merchant
 
   def test_acceptance_item
     # skip
-    data_location = {merchant: "./data/merchants.csv", item: "./data/fixtures/items.csv"}
+    data_location = {merchant: "./data/fixtures/merchants.csv", item: "./data/fixtures/items.csv", customer: "./data/fixtures/customers.csv"}
```

Comment:
Line is too long. [141/80]<br>Space inside { missing.<br>Space inside } missing.